docs/user-guide/install-nodejs-zos.md

```diff
@@ -12,7 +12,11 @@ Before you install Zowe&trade;, you must install IBM SDK for Node.js on the same
 
 The following Node.js versions are supported to run Zowe. See the [Hardware and software prerequisites](#hardware-and-software-prerequisites) section for the prerequisites that are required by Zowe. 
 
-The corresponding [IBM Knowledge Center for Node.js - z/OS](https://www.ibm.com/support/knowledgecenter/SSWLKB/welcome_nodesdk_family.html) lists all the prerequisites for Node.js. Some software packages, which might be listed as prerequisites there, are **NOT** required by Zowe. Specifically, you do **NOT** need to install Python, Make, Perl, or C/C++ runtime or compiler.  If you can run `node --version` successfully, you have installed the prerequisites required by Zowe.
+The corresponding [IBM Knowledge Center for Node.js - z/OS](https://www.ibm.com/support/knowledgecenter/SSWLKB/welcome_nodesdk_family.html) lists all the prerequisites for Node.js. Some software packages, which might be listed as prerequisites there, are **NOT** required by Zowe. Specifically, you do **NOT** need to install Python, Make, Perl, or C/C++ runtime or compiler. 
+
+Integrated Cryptographic Service Facility (ICSF) is required for Node.js to operate successfully on z/OS.  If you have not configured your z/OS environment for ICSF see [z/OS Cryptographic Services ICSF Administrators Guide](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.1.0/com.ibm.zos.v2r1.csfb300/toc.htm).  To see whether ICSF has been started check whether the started task `ICSF` is active.
```

Comment:
On some systems, ICSF is named CSF, so that's the STC name you need to look for.  It will only be one or the other.